ml_tools/eolearn/ml_tools/train_split.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tasks used for train set preparation.
+"""
+import numpy as np
+
+from eolearn.core import EOTask
+
+
+class TrainSplitTask(EOTask):
+    """ Randomly assigns each value to a class of values by generating a class mask.
+
+    Classes are defined by a list of cumulative probabilities, passed as the *bins* argument, the same way as the *bins*
+    argument in `numpy.digitize <https://docs.scipy.org/doc/numpy/reference/generated/numpy.digitize.html>`_. Valid
+    classes are enumerated from 1 onwards and if no_data_value is provided, all values equal to it get assigned to class
+    0.
```

Comment:
What I miss in the docs is description of _output_. In what respect will the EOPatch be different after the execution of this task.

Suggestion: After the execution of this task an EOPatch will have a new (FeatureType, new_name) feature where each pixel will have a value assigned representing the train, test and/or validation set.

Reply:
Done.